application/genesis/genesis.go

```diff
@@ -286,11 +286,11 @@ func (g *Genesis) storeContracts(ctx context.Context) error {
 
 		states = append(states, contracts.GetDepositGenesisContractState(
 			application.DefaultDistributionAmount,
-			int64(pulse.OfUnixTime(ApplicationIncentivesLockup)),
+			int64(pulse.OfUnixTime(ApplicationIncentivesUnholdDate-pulse.MinTimePulse)),
```

Comment:
I think you meant (pulse.OfUnixTime(ApplicationIncentivesUnholdDate)-pulse.MinTimePulse)